spec/eventbrite_sdk/resource_list_spec.rb

```diff
@@ -316,15 +395,15 @@ module EventbriteSDK
         list.next_page
 
         expect(request).to have_received(:get).with(
+          api_token: nil,
+          query: {},
           url: 'url',
-          query: { page: page_number },
-          api_token: nil
-        )
+        ).once
```

Comment:
I think #once is implicit - though I'm going off memory. Explicit is nice though in this case.

Reply:
Ahh, yes. On it.